src/spec/OpenApi.php

```diff
@@ -46,6 +47,7 @@ protected function attributes(): array
             'info' => Info::class,
             'servers' => [Server::class],
             'paths' => Paths::class,
+            'webhooks' => WebHooks::class,
```

Comment:
is there a specific reason why you introduce the "WebHooks" class and not implement this as a map of "string"->"PathItem" as described in the specification?

>  ... | ... | ...
>  -- | -- | --
> webhooks | Map[string, Path Item Object \| Reference Object] ] | The incoming webhooks that MAY be received as part of this API and that the API consumer MAY choose to implement. Closely related to the callbacks  feature, this section describes requests initiated other than by an API  call, for example by an out of band registration. The key name is a  unique string to refer to each webhook, while the (optionally  referenced) Path Item Object describes a request that may be initiated  by the API provider and the expected responses. An example is available.
>
> https://spec.openapis.org/oas/v3.1.0#openapi-object

Reply:
The `WebHooks` class is a copy, with some things changed, of the `Paths` class. Would the alternative just be wrapping a `Path::class` in an array, like `servers`?